development/file_formats/tscn.rst

```diff
@@ -1,84 +1,87 @@
 TSCN file format
 ================
 
-A :code:`.tscn` File format is the "Text SCeNe" file format and represents
-a single scene-tree inside Godot. TSCN files have the advantage of being
-nearly human-readable and easy for version control systems to manage. During
-import the TSCN files are compiled into binary :code:`.scn` files stored
-inside the .import folder. This reduces the data size and speed up loading.
+The TSCN (text scene) file represents a single scene tree inside
```

Comment:
Either "a TSCN file" or "the TSCN file format", saying "the TSCN file" implies only one file.